crates/ruff/src/rules/perflint/snapshots/ruff__rules__perflint__tests__PERF203_PERF203.py.snap

```diff
@@ -0,0 +1,31 @@
+---
+source: crates/ruff/src/rules/perflint/mod.rs
+---
+PERF203.py:2:5: PERF203 Try..except blocks can have significant overhead. Avoid using them inside of a loop.
+  |
+1 |   for i in range(10):
+2 |       try: # PERF203
+  |  _____^
+3 | |         print(f"{i}")
+4 | |     except:
```

Comment:
Should we only highlight the `except` header? Highlighting the whole `try` statement can create very large source frames

Reply:
Done - I brought that up in the PR description, wasn't sure if we should anchor on `try` or `except`. Thanks for the clarity!